lib/ConstExtract/ConstExtract.cpp

```diff
@@ -489,14 +500,45 @@ void writeAttributes(
   });
 }
 
+void writeResultBuilderInformation(llvm::json::OStream &JSON,
+                                   const swift::NominalTypeDecl *TypeDecl,
+                                   const swift::VarDecl *VarDecl) {
+  if (auto *attr = VarDecl->getAttachedResultBuilder()) {
+    JSON.attributeObject("resultBuilder", [&] {
+      JSON.attribute("type", toFullyQualifiedTypeNameString(attr->getType()));
+      writeFileInformation(JSON, VarDecl->getASTContext(), attr);
+    });
+
+    return;
+  }
+
+  for (ProtocolDecl *Decl :
+       TypeDecl->getLocalProtocols(ConformanceLookupKind::All)) {
+    for (auto Member : Decl->getMembers()) {
+      if (auto *VD = dyn_cast<swift::VarDecl>(Member)) {
+        if (auto *attr = VD->getAttachedResultBuilder()) {
+          if (VD->getName() == VarDecl->getName()) {
```

Comment:
@artemcm - Is checking equality by identifier the right thing to do here? It properly filtered in my testing.

Reply:
Yeah, those are uniqued by the ASTContext, so this should be safe. 👍🏼